docs/minikube.md

```diff
@@ -0,0 +1,74 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
```

Comment:
Curious @michelleN, are you reading it from https://github.com/rodcloutier/draft/blob/e4f801b79b755ca7283a7cfefbe5c0767153c99e/docs/minikube.md or elsewhere? The links work fine for me!

This line along with line 73 make up what's known as a reference-style link in Markdown and is correct syntax. I use it all the time.